src/torch_ucc_init_oss.cpp

```diff
@@ -1,4 +1,13 @@
 #include <torch/python.h>
+#include "torch_ucc.hpp"
 
-PYBIND11_MODULE(TORCH_EXTENSION_NAME, m) {
+using namespace c10d;
+
+extern "C" C10_EXPORT c10::intrusive_ptr<ProcessGroup> createProcessGroupUCCForNCCL(
```

Comment:
minor - might be good to use a more generic function name 
```suggestion
extern "C" C10_EXPORT c10::intrusive_ptr<ProcessGroup> createProcessGroupUCC(
```

Reply:
Changed